tests/auth_tests/nomigrations/nomigrations/settings.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+SECRET_KEY = 'django_auth_tests_secret_key'
+
+INSTALLED_APPS = [
+    'django.contrib.auth.apps.BaseAuthConfig',
+    'django.contrib.contenttypes',
+]
+
+MIGRATION_MODULES = {'auth': None}
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
Is it fine to use SQLite no matter what database the rest of the tests are running with? I can't remember a precedent for this, though I don't have an alternative.